### PR TITLE
Removed downscaling, which made the image quality worse in the video …

### PR DESCRIPTION
**Pull request by evekeen**
A naive attempt to fix image quality for imported tasks: https://github.com/openvinotoolkit/cvat/issues/4377

### CODE REVIEW

**Comment by nmanovic** [comment]
@Marishka17 , @azhavoro , please look at comment.

**Comment by evekeen** [comment]
That's actually a nasty bug, which made the whole dataset look incorrect, since the work was done with down-scaled images, while exported dataset is not downscaled. And objects can be in different locations because of the quality loss

**Comment by Marishka17** [comment]
@evekeen, Hello, thanks for the contribution. I'm not sure this is the correct solution.
I was able to reproduce the error when creating a task with a *.mov video file using the use_zip_chunks=True flag.
In this case, the `ZipCompressedChunkWriter` class is used to create a compressed chunk on the server.
If we create a backup of this task and import it, the `Mpeg4CompressedChunkWriter` will be used on the server.
I guess the problem with different quality is in this.

**Comment by evekeen** [comment]
@Marishka17 how do you suggest to fix  that?

**Comment by Marishka17** [comment]
As far as I remember the solution in [PR](https://github.com/openvinotoolkit/cvat/pull/3056)  was implemented without saving `use_zip_chunks` in `task.json`. When a task is creating [this](https://github.dev/openvinotoolkit/cvat/blob/b7e881f6c13491b600d19b82d89387ce0fbcf3aa/cvat/apps/engine/task.py#L457) check defines a chunk type, but in [buckup](https://github.dev/openvinotoolkit/cvat/blob/b7e881f6c13491b600d19b82d89387ce0fbcf3aa/cvat/apps/engine/backup.py#L507-L508) this option is missing.
 So I guess we need to save this metadata when creating a task backup.

**Comment by Marishka17** [comment]
@azhavoro, Do you have other suggestions?

**Comment by evekeen** [comment]
@Marishka17 I've just tried settings that option in my `task.json`, and it was ignored on import:
`WARNING cvat.server: the following keys are dropped {'use_zip_chunks'}`
Moreover, I see in the code we actually set `use_zip_chunks` programmatically [during the import](https://github.dev/openvinotoolkit/cvat/blob/b7e881f6c13491b600d19b82d89387ce0fbcf3aa/cvat/apps/engine/backup.py#L507)

**Comment by evekeen** [comment]
I've supported the `use_zip_chunks` option.
Also, I suggest keeping the initial change of downscaling, since I don't see a reason why downscale

File: cvat/apps/engine/backup.py [hunk 1]
```diff
@@ -504,8 +505,9 @@ def _write_data(zip_object):
         else:
             uploaded_files = _write_data(self._file)
 
-        data['use_zip_chunks'] = data.pop('chunk_type') == DataChoice.IMAGESET
+        use_zip_chunks = data.pop('use_zip_chunks', False) or data.pop('chunk_type') == DataChoice.IMAGESET
         data = data_serializer.data
+        data['use_zip_chunks'] = use_zip_chunks
```
Comment (Marishka17) [inline cvat/apps/engine/backup.py:510]: Sorry for my mistake. But I've taken a look at this one more time and these lines will be enough to fix. 
```suggestion
        use_zip_chunks = data.pop('chunk_type') == DataChoice.IMAGESET
        data = data_serializer.data
        data['use_zip_chunks'] = use_zip_chunks
```

Comment (Marishka17) [inline cvat/apps/engine/backup.py:510]: Sorry for my mistake. But I've taken a look at this one more time and these lines will be enough to fix. 
```suggestion
        use_zip_chunks = data.pop('chunk_type') == DataChoice.IMAGESET
        data = data_serializer.data
        data['use_zip_chunks'] = use_zip_chunks
```

Comment (evekeen) [reply cvat/apps/engine/backup.py:510]: Why? I have `chunk_type=video` there

Comment (Marishka17) [reply cvat/apps/engine/backup.py:510]: So, If you keep only this changes you have `chunk_type=video` only if you didn't select `use zip chunks`. If you selected `use_zip_chunks` option `chunk_type` is `imageset`. 
In which case do you have `chunk_type=video`? Could you please check one more time?

**Comment by Marishka17** [comment]
Hi, @evekeen, thanks for the contribution! Are you still going to contribute to us? If so, could you please resolve conflicts in this PR?

**Comment by Marishka17** [comment]
@azhavoro, Could you please take a look at the PR? I've resolved conflicts and updated it. PR should resolve issue #4377

**Review by azhavoro — APPROVED** [review]

**Comment by nmanovic** [comment]
@evekeen , it looks like most parts of your PR has been merged. I see now changes in the PR. Thank you for the contribution.

**Pull request closed without merging**